test/lit.cfg

```diff
@@ -151,3 +151,8 @@ else:
 # POSIX runtime feature
 if config.enable_posix_runtime:
   config.available_features.add('posix-runtime')
+
```

Comment:
Actually while we are here you could add `no-darwin` and `not-linux` features too. The tests are written such that it should really be `REQUIRE: not-darwin` rather than `REQUIRE: linux`